pkg/mig/mode/nvml.go

```diff
@@ -47,21 +47,21 @@ func NewMockNvmlMigModeManager(nvml nvml.Interface) Manager {
 
 func (m *nvmlMigModeManager) IsMigCapable(gpu int) (bool, error) {
```

Comment:
Will there be a follow up to use the equivalent functionality from `go-nvlib`?

Reply:
It's probably a good idea to audit this code to see what can be used from go-nvlib (or added to go-nvlib as necessary). This code predates go-nvlib so I'm sure there's lots of things that could be simpified now.